autospec/build.py

```diff
@@ -240,3 +240,16 @@ def package(filemanager, mockconfig, mockopts, cleanup=False):
         exit(1)
 
     parse_build_results(download_path + "/results/build.log", returncode, filemanager)
+
+    files = os.listdir('{}/results'.format(download_path))
+
+    os.makedirs('{}/results/srpm'.format(download_path))
+    os.makedirs('{}/results/debuginfo'.format(download_path))
+
+    # organize output files so it's more clear what to use for mixes
+    for f in files:
+        if ".src.rpm" in f:
+            print('copying {} to srpm')
```

Comment:
Autospec's output is already extremely spammy, mind taking this out?

Reply:
Oh yeah totally ^